tokens/_aliases.yml

```diff
@@ -1,93 +1,114 @@
 aliases:
 
-  # Mozilla Brand Colors
+  # Colors
 
-  brand-primary: '{!color-black}'
-  brand-secondary: '{!color-white}'
-  brand-neon-blue: '#00ffff'
-  brand-lemon-yellow: '#fff44f'
-  brand-warm-red: '#ff4f5e'
-  brand-neon-green: '#54ffbd'
-  brand-dark-purple: '#6e008b'
-  brand-dark-green: '#005e5e'
-  brand-dark-blue: '#00458b'
-
-  # UI Colors
-
-  color-black: '#000000'
+  color-black: '#20123a'
```

Comment:
There's only a subtle difference here, but it's worth calling out that this variable is used extensively. I'm assuming it's expected that all primary headlines, text and UI elements will use this new value, as opposed to true black?